go.mod

```diff
@@ -4,12 +4,12 @@ require (
 	github.com/Masterminds/squirrel v0.0.0-20190107164353-fa735ea14f09
 	github.com/VividCortex/mysqlerr v0.0.0-20170204212430-6c6b55f8796f
 	github.com/argoproj/argo-workflows/v3 v3.3.10
-	github.com/aws/aws-sdk-go v1.42.50
```

Comment:
since you updated `go.mod`, you'll need to update the license CSVs. you can do that by simply running `make all` in `backend/`

Reply:
resolved this in https://github.com/opendatahub-io/data-science-pipelines/pull/56